tests/test_alchemy.py

```diff
@@ -287,11 +278,20 @@ class SQLAlchemySessionFactoryTestCase(unittest.TestCase):
     def test_create_get_session_from_sqlalchemy_session_factory(self):
         SessionGetterFactory.create()
         self.assertEqual(SessionGetterFactory._meta.sqlalchemy_session, models.session)
+        # Let's try again, since we now have a session set from the factory, and ensure we won't
+        # try to validate again
```

Comment:
Up to you.
```suggestion
        # Reuse the session obtained from sqlalchemy_session_factory.
```

Reply:
Done in [27f3a77](https://github.com/FactoryBoy/factory_boy/pull/927/commits/27f3a77ee634b7863e20f335a5e413d21db82a2c)